engine/gamesys/src/gamesys/components/comp_sprite.cpp

```diff
@@ -111,6 +111,7 @@ namespace dmGameSystem
 
     static const dmhash_t SPRITE_PROP_CURSOR = dmHashString64("cursor");
     static const dmhash_t SPRITE_PROP_PLAYBACK_RATE = dmHashString64("playback_rate");
+    static const dmhash_t SPRITE_PROP_ANIM_ID = dmHashString64("animation_id");
```

Comment:
For consistency with gui, I prefer "animation"
https://defold.com/ref/stable/gui/#gui.get_flipbook:node

Reply:
Do you mean `animation` instead of `animation_id` ?

Reply:
fixed